include/mxnet/ndarray.h

```diff
@@ -744,6 +748,11 @@ class NDArray {
   NDArray MKLDNNDataReshape(const TShape &shape) const;
 #endif
 
+#if MXNET_USE_NGRAPH == 1
+    // create and return tensor_view with this ndarray mem
+    std::shared_ptr<ngraph::runtime::TensorView> &create_tensor_view();
+#endif
```

Comment:
where is the method called?

Reply:
This method is called in the code in `3rdparty/ngraph-mxnet-bridge`.